containers/ftp/ftp_files.rb

```diff
@@ -0,0 +1,198 @@
+load './ftp_patch.rb'
+load './utils.rb'
+
+class FtpFiles
+  include Utils
+  attr_reader :logger
+
+  def initialize(logger)
+    @logger = logger || Logger.new(STDOUT)
+  end
+
+  #
+  # upload_file
+  #
+  #  == Parameters:
+  # @param uri [URI] the uri for where to FTP the file to
+  # @param local_file
+  # @param options [Hash]
+  # - retry: retries by default, so this is true by default
+  # - retry_wait: defaults to 10 seconds
+  # - retry_max: when there is retry, defaults to 6 tries
+  # - md5: will write an md5 by default, defaults to true
```

Comment:
I think as a generic FTP task, writing the MD5 file is an unexpected default. I would think that if the task doesn't specify, the task would FTP only the source file to the given destination.

Reply:
I know that for remix and the exchange that both used this code, writing the md5 is the default and more common case, and that's been that way for many years. That said, I don't care, and it is an extra to some generic/base case; I can pass it in.